graphene/relay/connection.py

```diff
@@ -50,8 +50,9 @@ class Meta:
         abstract = True
 
     @classmethod
-    def __init_subclass_with_meta__(cls, node=None, name=None, **options):
-        _meta = ConnectionOptions(cls)
+    def __init_subclass_with_meta__(cls, node=None, name=None, _meta=None, **options):
+        if not _meta:
+            _meta = ConnectionOptions(cls)
```

Comment:
Maybe
```python
    def __init_subclass_with_meta__(cls, node=None, name=None, connection_options=None, **options):
        _meta = connection_options or ConnectionOptions(cls)
```
and have a test for this.

Reply:
I updated to add a unit test. I went with the _meta kwarg over connection_options because then it is the same name flowing through the entire process chain. 

If you think it makes more sense to change it to connection_options I will do that, I just think it feels weird to change the naming convention.

Reply:
I just looked at this function and made the common comment. It should be fine if this repo went with the convention.